### DISCUSSION TITLE
独立音量控制和更多级音量调节

### DISCUSSION THREAD

**Discussion by alsjiop** [ywmoyue/biliuwp-lite#630] [feature]
### 问题描述

比如打开一个音乐视屏,然后再打开另一个其他视屏,但是音乐视屏的声音可能太大或太小,会无法听清或覆盖掉其他视屏的声音

### 描述解决方案

能不能给各个视屏页面的音量控制设置成独立的音量控制,然后增加更多级的音量大小,因为某些视屏就算拉满也很小声

### 代码贡献

否

**Comment by ywmoyue** [comment]
独立音量的话试试这个设置项
![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/340a27d0-71a5-4ec3-aeef-12c2fff6b94f)

**Reply by alsjiop** [reply]
感谢回复，有用😋
